### PR TITLE
Adds two new templates

### PR DESCRIPTION
**Pull request by scarolan**
The two tracks are:

persist-shell-session
instruqt-with-terraform

### CODE REVIEW

File: instruqt-with-terraform/track.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  1. User runs terraform commands to configure resources in AWS.
+
+  2. User configures a SaaS provider and runs terraform to manage resources on the SaaS platform.
+
+  3. Background setup using terraform that happens before your user logs in. Look in the **track_scripts/setup-workstation** script for an example of terraform that runs during setup.
+icon: https://storage.googleapis.com/instruqt-frontend/img/tracks/default.png
+tags:
+- terraform
+- saas
+owner: instruqt
```
Comment (matthisk) [inline instruqt-with-terraform/track.yml:21]: I think it is confusing if not all templates are owned by the templates organization. Is there any specific reason why you decided to put these in `instruqt`?

File: instruqt-with-terraform/02-intro-to-terraform/assignment.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+terraform version
+terraform -help
+terraform init
+```
+
+The **terraform init** command downloads any required providers and modules to your workstation. These are stored in the hidden **.terraform** directory.
+
+Nice work. Your Terraform workspace is ready for use.
+
+Click on the **Check** button below to continue.
```
Comment (matthisk) [inline instruqt-with-terraform/02-intro-to-terraform/assignment.md:61]: When there is no check script, the button will actually read **Next** ;)

File: instruqt-with-terraform/track.yml (hunk 1 above)
Comment (scarolan) [reply instruqt-with-terraform/track.yml:21]: I wasn't sure where to put them.  will move them over to the instruqt-demos org.

**Comment by scarolan** [comment]
Closing because these tracks don't belong in templates.

**Pull request closed without merging**